spec/bridges/hubee_cert_dc_bridge_spec.rb

```diff
@@ -0,0 +1,88 @@
+RSpec.describe HubEECertDCBridge do
+  subject(:hubee_cert_dc_bridge) { described_class.new(authorization_request).perform }
+
+  let(:hubee_api_client) { instance_double(HubEEAPIClient) }
+
+  let(:authorization_request) { create(:authorization_request, id: 2, organization_id: organization.id, data: authorization_request_data, last_validated_at: 'Thu, 18 Jul 2024 14:00:55.500378000 CEST +02:00', updated_at: 'Thu, 18 Jul 2024 14:00:55.500378000 CEST +02:00', linked_token_manager_id: nil) }
+  let(:authorization_request_data) { { 'administrateur_metier_email' => 'admin@yopmail.com', 'administrateur_metier_phone_number' => '01.23.45.67.89' } }
+
+  let(:organization) { create(:organization, id: 3, siret: '21920023500014') }
+
+  let(:organization_payload) { build(:hubee_organization_payload) }
+  let(:subscription_response) { build(:hubee_subscription_response_payload, :cert_dc) }
```

Comment:
Ce setup me parait très compliqué: aucune valeur n'est vraiment utilisée ici, ou alors c'est caché et donc compliqué à comprendre. Je vais proposer quelque chose :

```suggestion
  let(:authorization_request) { create(:authorization_request, :hubee_cert_dc, organization:) }

  let(:organization) { create(:organization, :with_insee_data) } # je ne suis pas sûr que ce trait existe mais ça peut se faire du coup

  let(:organization_payload) { build(:hubee_organization_payload, organization: ) }
  let(:subscription_response) { build(:hubee_subscription_response_payload, :cert_dc, authorization_request:) }
```

Et ici on modifie les factories hubee. Exemple avec la première:

```ruby
factory :hubee_organization_payload do
  transient do
    organization { nil }
  end

  after(:build) do |hubee_organization_payload, evaluator|
    if evaluator.organization.present?
      hubee_organization_payload[:companyRegister] = evaluator.organization.siret
      hubee_organization_payload[:branchCode] = evaluator.organization.postal_code
    end
  end
end
```

Reply:
Je ne suis pas certaine de saisir 

> `let(:organization) { create(:organization, :with_insee_data) }` # je ne suis pas sûr que ce trait existe mais ça peut se faire du coup

Quand je lance organization dans mon debugger avec la ligne 
`let(:organization) { create(:organization, id: 3, siret: '21920023500014') }`, j'ai bien une payload_insee incluse.